nucleus/pom.xml

```diff
@@ -1482,6 +1482,27 @@ Parent is ${project.parent}</echo>
                 </pluginManagement>
             </build>
         </profile>
+        
+        <profile>
+            <id>add-source</id>
```

Comment:
Why different from `embed-source` in packager?

Reply:
One profile just adds the source jars, but leaves them where they are.  

The packager profile embeds them into the existing jars, which is a somewhat distinct operation.

Reply:
Ok, so you don't need both of them to work together. @Cousjava seem to suggest there's a way to achieve that already, though.